refactoring.api/src/org/netbeans/modules/refactoring/api/RefactoringSession.java

```diff
@@ -116,79 +108,66 @@ private Problem reallyDoRefactoring(boolean saveAfterDone) {
+                        try {
+                            commit.commit();
+                        } finally {
                             progressProvider.removeProgressListener(progressListener);
                         }
+                    } else {
+                        commit.commit();
                     }
-                    if (wrapper != null) {
-                        unsetWrappers(commit, wrapper);
+                    if (undoableWrapper != null) {
+                        undoableWrapper.setActive(false, null);
                     }
+                });
+                if (undoableWrapper != null) {
+                    undoableWrapper.close();
                 }
-                if (wrapper != null) {
-                    wrapper.close();
-                }
-                for (Transaction commit : commits) {
-                    SPIAccessor.DEFAULT.sum(commit);
-                }
+                commits.forEach(commit -> SPIAccessor.DEFAULT.sum(commit));
             }
             if (saveAfterDone) {
                 LifecycleManager.getDefault().saveAll();
-                for (DataObject dob:DataObject.getRegistry().getModified()) {
-                    SaveCookie cookie = dob.getCookie(SaveCookie.class);
+                for (DataObject dataObject : DataObject.getRegistry().getModified()) {
+                    SaveCookie cookie = dataObject.getLookup().lookup(SaveCookie.class);
                     try {
                         cookie.save();
                     } catch (IOException ex) {
                         Exceptions.printStackTrace(ex);
                     }
                 }
             }
-            for (RefactoringElementImplementation fileChange:SPIAccessor.DEFAULT.getFileChanges(bag)) {
-                if (fileChange.isEnabled()) {
-                    fileChange.performChange();
-                }
-            }
+            SPIAccessor.DEFAULT.getFileChanges(bag).stream().filter(fileChange -> fileChange.isEnabled()).forEachOrdered(fileChange -> fileChange.performChange());
             fireProgressListenerStep();
         } finally {
             fireProgressListenerStop();
             if (realcommit) {
                 undoManager.addItem(this);
                 undoManager.transactionEnded(false, this);
-                realcommit=false;
+                realcommit = false;
             }
         }
         Logger timer = Logger.getLogger("TIMER.RefactoringSession");
         if (timer.isLoggable(Level.FINE)) {
-            time = System.currentTimeMillis() - time;
-            timer.log(Level.FINE, "refactoringSession.doRefactoring", new Object[] { description, RefactoringSession.this, time } );
+            final long timeTaken = System.currentTimeMillis() - start;
+            timer.log(Level.FINE, "refactoringSession.doRefactoring", new Object[]{description, RefactoringSession.this, timeTaken});
```

Comment:
Considering most of the PR fixes whitespaces, there should be whitespaces before/after { to separate from type and array content, and before }.

Reply:
I don't know, what a PR fix is. But to separate things, to get a better overview, seems to be a good idea.